services/tracing/migrations/20230821113939_initial_tracing.up.sql

```diff
@@ -15,7 +15,7 @@ CREATE TABLE steps (
     tracing_id VARCHAR(255) NOT NULL,
     step_index BIGINT NOT NULL,
     steps_remaining BIGINT NOT NULL,
-    start_time timestamptz NOT NULL,
+    start_time timestamptz(3) NOT NULL,
```

Comment:
what does the 3 do?

Reply:
Save the timestamp to 3 decimals of the second. So milliseconds. 
I noticed that the tracing service only outputs seconds, and I couldn't get it fixed. This is pretty much a remnant.